polars_unpack/unpack.py

```diff
@@ -822,6 +760,85 @@ class UnknownDataTypeError(Exception):
     """When an unknown/unsupported datatype is encountered."""
 
 
+@pl.api.register_dataframe_namespace("json")
+@pl.api.register_lazyframe_namespace("json")
```

Comment:
Decided to go with the decorators, it feels cleaner.

Reply:
Definitely. For some reason I thought I needed to ditch it due to the double api registration xD